libs/application_runner/lib/mongo_storage.ex

```diff
@@ -73,12 +71,51 @@ defmodule ApplicationRunner.MongoStorage do
   # DATA #
   ########
 
+  @doc """
+  Creates documents from the specified `docs`
+  """
+  def create_docs(env_id, coll, docs) do
+    Logger.debug("#{__MODULE__} create_docs for env_id: #{env_id}, coll: #{coll}, docs: #{inspect(docs)}")
+
+    decoded_docs = Enum.map(docs, fn doc -> decode_ids(doc) end)
+
+    env_id
+    |> mongo_instance()
+    |> Mongo.insert_many(coll, decoded_docs)
+    |> case do
+      {:error, err} ->
+        TechnicalError.mongo_error_tuple(err)
+
+      {:ok, res} ->
+        # TODO: Think about something to return
+        {:ok, %{}}
+    end
+  end
+
+  def create_docs(env_id, coll, docs, session_uuid) do
```

Comment:
You should use the Mongo method name:
```suggestion
  def insert_many(env_id, coll, docs) do
    Logger.debug("#{__MODULE__} create_docs for env_id: #{env_id}, coll: #{coll}, docs: #{inspect(docs)}")

    decoded_docs = Enum.map(docs, fn doc -> decode_ids(doc) end)

    env_id
    |> mongo_instance()
    |> Mongo.insert_many(coll, decoded_docs)
    |> case do
      {:error, err} ->
        TechnicalError.mongo_error_tuple(err)

      {:ok, res} ->
        # TODO: Think about something to return
        {:ok, %{}}
    end
  end

  def insert_many(env_id, coll, docs, session_uuid) do
```